scripts/run_tests.sh

```diff
@@ -17,6 +17,10 @@ fi
 FILES=$(go list ./... | grep -v github.com/iotaledger/wasp/contracts/wasm)
 ${GO_EXECUTABLE} clean -testcache
 
+make compile-solidity
+make wasm
+make install
```

Comment:
```suggestion
make wasm
make install
```

I think make install already does the solidity bit